src/app/components/BodyHandsPage.tsx

```
@@ -258,29 +267,34 @@ export function BodyHandsPage({
-                className={`font-din-arabic text-sm transition-colors duration-300 ${
-                  selectedFilter === "all"
-                    ? "text-black border-b border-black"
-                    : "text-black/40 hover:text-black/70"
-                }`}
+                className={`font-din-arabic text-sm transition-colors duration-300 ${selectedFilter === "all"
+                  ? "text-black border-b border-black"
+                  : "text-black/40 hover:text-black/70"
+                  }`}
                 style={{ letterSpacing: "0.15em" }}
               >
                 All products
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)